app/services/save_practice_service.rb

```diff
@@ -54,6 +54,11 @@ def save_practice
       rescue_method(:update_practice_awards)
       rescue_method(:update_category_practices)
       rescue_method(:crop_resource_images)
+      if updated
```

Comment:
There are a lot of things in the PE that update associated tables i..e, resources, metrics ..etc that don't actually update the practices table last_updated field.  The way I interpreted Blaine's email is that she wants to use this list to tell who has recently updated any part of their practice.  I guess we could use another column i.e., "practice_pages_updated" and display that in the view instead of the time_stamp column "updated_at" which really is just for the practice table... ?? What do you think?

Reply:
After thinking about this...  :)  I'll just create a new column in the practices table for this... and update it whenever we call save_practice and updated is true...  best to not mess with the updated_at field... since it is a system timestamp for the record.. and could throw auditing off if that ever became an issue...  I'll let you know when it's updated and ready for review..

Reply:
sounds good to me! 👍